docs/static/arcsight-module.asciidoc

```diff
@@ -104,32 +100,19 @@ the EB event stream.
 
 image::static/images/arcsight-diagram-adp.svg[ArcSight Event Broker architecture]
 
-[[arcsight-requirements-eventbroker]]
-===== Requirements
-
-* These instructions assume you have the Elastic Stack (Logstash, Elasticsearch,
-Kibana) already installed. The products you need are
-https://www.elastic.co/downloads[available to download] and easy to install. The
-Elastic Stack 5.6 or higher is required for this module.
-* The Elastic Stack is running locally with default ports exposed, namely
-Elasticsearch as "localhost:9200" and Kibana as "localhost:5601".
-* By default, the Logstash ArcSight module consumes from the EB "eb-cef" topic.
+By default, the Logstash ArcSight module consumes from the EB "eb-cef" topic.
```

Comment:
while you are in here... nitpick: `EB` is not a standard abbreviation , and IMO should be avoided.

Reply:
++